README.md

```diff
@@ -1,7 +1,7 @@
 # SPV Wallet Toolbox
 
 This is the main documentation for the SPV Wallet ecosystem. Here you can find all the information about it and how to use it.
-
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
The text before the link (staring "This is the main documentation") can make a reader confused. Because, for example, I don't know if the content under the link is the main documentation or the content of this repo.

Reply:
Moved to the bottom